src/__tests__/xhr.js

```diff
@@ -0,0 +1,72 @@
+import { xhr } from '../send-request'
```

Comment:
We already have a test file for send-request, why create a new one? This breaks the invariant of test file being named the same as module under test.

Reply:
"breaks the invariant of test file being named the same as module under test" moved to follow local style